etna/search/views.py

```diff
@@ -655,6 +716,17 @@ class FeaturedSearchView(BaseSearchView):
     api_method_name = "search_all"
     form_class = FeaturedSearchForm
     template_name = "search/featured_search.html"
+    title_base = "All results"
+
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
+        data = super().get_datalayer_data(request)
+        total_count = 0
+        for result in self.api_result["responses"]:
+            total_count += result["hits"]["total"]["value"]
```

Comment:
`get_buckets` in this view implements `bucket.result_count = response["hits"]["total"]["value"]`
Can implement an attribute `featured_search_total_count` and use `self.featured_search_total_count += bucket.result_count` to calculate the count instead?